src/components/ExportDataModal.js

```diff
@@ -17,9 +17,24 @@ import * as Utils from 'src/libs/utils'
 import validate from 'validate.js'
 
 
-const InfoTile = ({ infoStyle, content, iconName }) => {
-  return div({ style: { ...infoStyle, display: 'flex', alignItems: 'center' } }, [
-    icon(iconName, { size: 36, style: { flex: 'none', marginRight: '0.5rem' } }),
+const warningStyle = {
+  border: `1px solid ${colors.warning(0.8)}`, borderLeft: 'none', borderRight: 'none',
+  backgroundColor: colors.warning(0.15),
+  padding: '1rem 1.25rem', margin: '0 -1.25rem',
+  fontWeight: 'bold', fontSize: 12
+}
+const errorStyle = {
+  ...warningStyle,
+  border: `1px solid ${colors.danger(0.8)}`,
+  backgroundColor: colors.danger(0.15)
+}
+
+const InfoTile = ({ type = 'warning', content }) => {
+  const [style, shape, color] = Utils.switchCase(type,
+    ['error', () => [errorStyle, 'error-standard', colors.danger()]],
+    [Utils.DEFAULT, () => [warningStyle, 'warning-standard', colors.warning()]])
```

Comment:
Why not just:
```suggestion
const InfoTile = ({ isError = false, content }) => {
  const [style, shape, color] = isError ? 
    [errorStyle, 'error-standard', colors.danger()] :
    [warningStyle, 'warning-standard', colors.warning()]
```
and then make the corresponding prop change below?

Reply:
Yup. That's the logical next refactor. Thanks. The `switchCase` made sense to me right up until I added the default value for `type`. And you're absolutely right... this is a binary property right now. No need to add unused complexity.